miden-lib/asm/note_scripts/P2ID.masm

```diff
@@ -52,23 +63,24 @@ proc.add_note_assets_to_account
     drop dropw drop
```

Comment:
At the end of the `add_note_assets_to_account` procedure we have internal, specific data on the stack, so it will be a little bit strange to return `# => [data_ptr, ASSET, end_ptr]` from this proc and then immediately drop them after. So IMO it is better to leave it as it is.